lang/de_DE.lang

```diff
@@ -58,93 +70,102 @@ achievement.myst.symbol.desc=Finde ein Symbol
-myst.symbol.ColorFog.name=Nebelfarbe
-myst.symbol.ColorFogNat.name=Natürliche Nebelfarbe
-myst.symbol.ColorFoliage.name=Blattwerkfarbe
-myst.symbol.ColorFoliageNat.name=Natürliche Blattwerkfarbe
-myst.symbol.ColorGrass.name=Grasfarbe
-myst.symbol.ColorGrassNat.name=Natürliche Grasfarbe
-myst.symbol.ColorHorizon.name=Sonnenuntergangsfarbe
-myst.symbol.ColorSky.name=Himmelsfarbe
-myst.symbol.ColorSkyNat.name=Natürliche Himmelsfarbe
-myst.symbol.ColorSkyNight.name=Nachthimmelsfarbe
-myst.symbol.ColorWater.name=Wasserfarbe
-myst.symbol.ColorWaterNat.name=Natürliche Wasserfarbe
-myst.symbol.CryForm.name=Kristallformationen
-myst.symbol.DenseOres.name=Dichte Erze
-myst.symbol.Dungeons.name=Verliese
-myst.symbol.EnvAccel.name=Beschleunigt
-myst.symbol.EnvExplosions.name=Spontane Explosionen
-myst.symbol.EnvLightning.name=Blitz und Donner
-myst.symbol.EnvMeteor.name=Meteore
-myst.symbol.EnvScorch.name=Versengte Oberfläche
-myst.symbol.Flat.name=Flache Welt
-myst.symbol.FloatIslands.name=Schwebende Inseln
-myst.symbol.GenSpikes.name=Stacheln
-myst.symbol.HugeTrees.name=Riesige Bäume
-myst.symbol.LakesDeep.name=Untergrundseen
-myst.symbol.LakesSurface.name=Oberflächenseen
-myst.symbol.LightingBright.name=Helles Licht
-myst.symbol.LightingDark.name=Dunkles Licht
-myst.symbol.LightingNormal.name=Normales Licht
-myst.symbol.Mineshafts.name=Minenschächte
-myst.symbol.ModClear.name=Modifizierungen entfernen
-myst.symbol.ModGradient.name=Farbverlauf
-myst.symbol.MoonDark.name=Dunkler Mond
-myst.symbol.MoonNormal.name=Normaler Mond
-myst.symbol.NetherFort.name=Netherfestung
-myst.symbol.NoHorizon.name=Grenzenloser Himmel
-myst.symbol.NoSea.name=Keine Seen
-myst.symbol.Obelisks.name=Obelisken
-myst.symbol.PvPOff.name=Kein PVP
-myst.symbol.Rainbow.name=Regenbogen
-myst.symbol.Ravines.name=Schluchten
-myst.symbol.Skylands.name=Himmelslande
-myst.symbol.StarFissure.name=Sternenkluft
-myst.symbol.StarsDark.name=Dunkle Sterne
-myst.symbol.StarsEndSky.name=Endersternenhimmel
-myst.symbol.StarsNormal.name=Normale Sterne
-myst.symbol.StarsTwinkle.name=Funkelnde Sterne
-myst.symbol.Strongholds.name=Festungen
-myst.symbol.SunDark.name=Dunkle Sonne
-myst.symbol.SunNormal.name=Normale Sonne
-myst.symbol.Tendrils.name=Ranken
-myst.symbol.TerModSpheres.name=Sphären
-myst.symbol.TerrainAmplified.name=Zerklüftete Welt
-myst.symbol.TerrainEnd.name=Inselwelt
-myst.symbol.TerrainNether.name=Höhlenwelt
-myst.symbol.TerrainNormal.name=Standardwelt
-myst.symbol.WeatherCloudy.name=Bewölkt
-myst.symbol.WeatherFast.name=Schnelles Wetter
-myst.symbol.WeatherNorm.name=Normales Wetter
-myst.symbol.WeatherOff.name=Kein Wetter
-myst.symbol.WeatherOn.name=Ewiges Wetter
-myst.symbol.WeatherRain.name=Ewiger Regen
-myst.symbol.WeatherSlow.name=Langsames Wetter
-myst.symbol.WeatherSnow.name=Ewiger Schnee
-myst.symbol.WeatherStorm.name=Ewiger Sturm
-myst.symbol.Villages.name=Dörfer
-myst.symbol.Void.name=Leere Welt
+mystsymbol.BioConGrid.name=Gittermäßige Biomverteilung
```

Comment:
See? Here.
The previous key was 
myst.symbol.BioConHuge.name
now the key is
mystsymbol.BioConHuge.name

You changed the key by accident.